docs/usage/delete.rst

```diff
@@ -17,6 +17,7 @@ Examples
 
     # see what would be deleted if delete was run without --dry-run
     $ borg delete -v --dry-run -a '*-May-*' /path/to/repo
+    $ borg delete --list --dry-run -a '*-May-*' /path/to/repo
```

Comment:
can you check whether we still need the example in 19 (and not just the one in 20)?